spatial-tracking-explainer.md

```diff
@@ -197,37 +197,58 @@ function onSessionStarted(session) {
+* A motion controller that has temporarily lost positional tracking but is still held, whose orientation continues to update at the last-known position relative to the viewer.
 
-### Viewer space
-Calls to `XRFrame.getViewerPose()` return an `XRViewerPose` object which contains the pose of the viewer along with the views to be rendered. Sometimes it is useful to have access to the `XRSpace` represented by the viewer directly, such when the developer wants to use it to compare locations against other `XRSpace` objects.
+This base tracking loss behavior enables developers to then build whatever tracking loss behavior their scenario requires.
 
-```js
-  let pose = xrFrame.getPose(preferredInputSource.gripSpace, xrSession.viewerSpace);
-  if (pose) {
-    // Calculate how far the motion controller is from the user's head
-  }
-```
+For viewer poses, it's common in VR experiences to let the world drag along with you during positional tracking loss, as this can be preferable to halting the experience. By continuing to render the experience with viewer poses even when `emulatedPosition` is true, this behavior can be obtained easily. In contrast, when building an AR experience where anchoring of rendered objects to the real world is more critical, developers can detect that the viewer pose's `emulatedPosition` has become false and stop rendering the main scene, falling back to an orientation-only warning.
+
+For input source poses, motion controllers may have their own inertial tracking that can continue updating orientation while positional tracking is lost. If using a controller primarily to target distant objects, developers may simply ignore `emulatedPosition` and continue to point using each frame's updated target ray as the user rotates the controller. If using a controller to do fine operations, such as painting at the controller's tip, developers may instead choose to stop painting when `emulatedPosition` becomes false, ensuring that only high-quality paint strokes are drawn.
+
+#### Tracking recovery
+As discussed above, during tracking loss, the viewer's pose will remain at their last-known position within each reference space, while continuing to incorporate orientation updates and any slight adjustments due to neck modeling. When the viewer then recovers positional tracking, developers may observe an instantaneous jump of the viewer pose. If the application continues to render during this time, the world will appear to drag along as the viewer moves around.
+
+When the viewer then recovers positional tracking, the origin of each `eye-level`, `floor-level`, `bounded` and `unbounded` reference space will continue to track the same physical location as it did before tracking was lost. Therefore, in the first frame where the viewer pose's `emulatedPosition` is false again, the developer will observe an abrupt jump in the viewer pose back to its tracked position.
```

Comment:
Does the phrasing cause misunderstandings in practice?

The `viewer` space precisely maps to the pose of the [viewer](https://immersive-web.github.io/webxr/#viewer).  Whichever one your brain binds "viewer" to, you are correct :smile:

Reply:
Maybe I'm looking for it to say "all viewer reference spaces' poses will remain"?  It's kind of a nit, though...

Reply:
Looking at the previous paragraph as well, I think one catch here is that I'm using the terms "viewer pose", "viewer recovers tracking", etc. to refer to one logical thing, where the viewer is and if it's tracked, that's then expressed in WebXR in two ways:
* The viewer pose (with emulatedPosition bool) returned by `getViewerPose(baseSpace)`
* The pose (with emulatedPosition bool) of the `viewer` reference space returned by `getPose(viewerSpace, baseSpace)`

I could be more specific throughout the explainer and say "`viewer` reference space" explicitly, but I'm not sure that excluding `getViewerPose` makes things more correct, since that's equally affected by all this tracking loss/recovery behavior too.